src/node/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ESNext",
     "module": "ESNext",
-    "lib": [ "ESNext" ],
+    "lib": [ "ESNext", "DOM" ],
```

Comment:
todo for myself: replace this with either smaller definition or use generated .d.ts types (preferred).